src/main/java/frc/robot/commands/Shooter_tuner.java

```diff
@@ -40,7 +40,7 @@ public void execute() {
   @Override
   public void end(boolean interrupted) {
 
-    shooter.setSetVolt(0);
+    shooter.setVoltage(0);
```

Comment:
should we set 0 voltage here or stopMotor here?

Reply:
Yup